doc/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,51 @@
+
+## Troubleshooting
+
+### Common Pitfalls
+#### Login profile files
+When the SSH slaves plugin connects to a slave, it does not run an interactive shell. 
+Instead it does the equivalent of your running `ssh slavehost command...` a few times, 
+eventually to run `ssh slavehost java -jar ...`. Exactly what happens on the slave as a result of this depends on the SSHD implementation, 
+but OpenSSH runs this with `bash -c command ...` (or whatever your login shell is.)
+
+This means some of your login profiles that set up your environment is not read by your shell. See [this post](http://stackoverflow.com/questions/216202/why-does-an-ssh-remote-command-get-fewer-environment-variables-then-when-run-man) for more details.
```

Comment:
"is" -> "are" 

("Some" indicates a likelihood of more than one, so it uses the plural form.)